src/plonesocial/core/browser/tiles/newpostbox.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from plone import api
+from plone.tiles import Tile
+from plonesocial.core.browser.stream import StreamBase
+
+
+class NewPostBoxTile(Tile, StreamBase):
+
+    index = ViewPageTemplateFile('templates/new-post-box-tile.pt')
+
+    def render(self):
+        return self.index()
+
+    def __call__(self):
+        if self.request.form:
+            view = api.content.get_view(
+                'stream',
```

Comment:
It could be a matter of naming.
I think that the tile I am replacing (https://github.com/ploneintranet/ploneintranet.theme/blob/a7ce8f5bdfc46055637c89019d89905454d37deb/src/ploneintranet/theme/browser/dashboard.py#L41)
took its name because the id of the form is new-post-box.

As far as I know the @@newpostbox.tile is just use here:
 - https://github.com/ploneintranet/ploneintranet.theme/blob/a7ce8f5bdfc46055637c89019d89905454d37deb/src/ploneintranet/theme/browser/templates/dashboard.pt#L25

If we want we can just change the name for the tile, the class and the template.

If you have a better name to suggest I will be pleased to modify the pull request.